server/liveblog/blogs/blogs.py

```diff
@@ -144,6 +147,7 @@ def on_created(self, docs):
 
     def find_one(self, req, **lookup):
         doc = super().find_one(req, **lookup)
+#         notify_the_owner(doc, app.config['CLIENT_URL'])
```

Comment:
if you have a  good reason to keep commented, it's better to add a note to let us know why it is important